packages/strapi-plugin-content-manager/admin/src/containers/EditPage/saga.js

```diff
@@ -1,13 +1,13 @@
-import { LOCATION_CHANGE } from 'react-router-redux';
+// import { LOCATION_CHANGE } from 'react-router-redux';
```

Comment:
Many comments, should we remove them ?

Reply:
No I need to combine the routers which is going to be made in another PR.

You will see comments like the following that explains it is going to be made:
```
  // TODO fix router (Other PR) <======
  // const loadDataWatcher = yield fork(takeLatest, GET_DATA, dataGet);
```

The app works great without canceling the sagas (they are being ejected when the component umount so it's not a huge issue).

To be honest I haven't looked at the router too closely since I'm going to remove the [react-router-redux](https://github.com/reactjs/react-router-redux) which is deprecated.

Reply:
No I need to combine the routers which is going to be made in another PR.

You will see comments like the following that explains it is going to be made:
```
  // TODO fix router (Other PR) <======
  // const loadDataWatcher = yield fork(takeLatest, GET_DATA, dataGet);
```

The app works great without canceling the sagas (they are being ejected when the component umount so it's not a huge issue).

To be honest I haven't looked at the router too closely since I'm going to remove the [react-router-redux](https://github.com/reactjs/react-router-redux) which is deprecated.